lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -202,8 +203,18 @@ function REPL( options ) {
 	// Define the current workspace:
 	setNonEnumerable( this, '_currentWorkspace', 'base' );
 
-	// Initialize an internal flag indicating whether a user is entering a multi-line command:
-	setNonEnumerable( this, '_multiline_mode', false );
+	// Initialize an internal object storing user settings:
+	setNonEnumerable( this, '_settings', {} );
+	setNonEnumerable( this._settings, 'autoCloseSymbols', true );
+
+	// Initialize an internal status object for controlling multi-line mode:
+	setNonEnumerable( this, '_multiline_mode', {} );
```

Comment:
Maybe we can also shorten then is to `_multiline`, rather than `multiline_mode`, now that it is an object with various properties.